app/views/sufia/admin/admin_sets/index.html.erb

```diff
@@ -27,6 +28,7 @@
                     <td><%= link_to admin_set.title.first, [sufia, :admin, admin_set] %></td>
                     <td><%= admin_set.create_date %></td>
                     <td><%= safe_join(admin_set.creator) %></td>
+                    <td><%= @presenter.total_items(admin_set) %></td>
```

Comment:
OK, got it. Code is working by placing `<td><%= controller.presenter_class.new(admin_set, current_ability).total_items %></td>` in view. However, I'm not sure how to account for this in the view test. Test keeps giving me `undefinded method 'presenter_class'`